### PR TITLE
feat: delete

### PR DESCRIPTION
**Pull request by StanislavKharkiv**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 postgres
 .DS_Store
-__pycache__
\ No newline at end of file
+__pycache__
+database/data
```
Comment (sviridovartem) [inline .gitignore]: add new line

File: database/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM postgres:latest
+FROM postgres:16.1
 ENV POSTGRES_PASSWORD=secret
```
Comment (sviridovartem) [inline database/Dockerfile]: Note: don't save any passwords in github
create .env file, and create .env.example file(you can push this file), update dockerfile to use .env file instead passwords and credentials

File: app/flask/models/tasks.py [hunk 3]
```diff
@@ -27,26 +27,47 @@ def __init__(self, conn, cursor) -> None:
             """
         )
 
+        self.cursor.execute(
+            "ALTER TABLE Tasks ADD COLUMN IF NOT EXISTS is_deleted BOOLEAN DEFAULT false;"
```
Comment (sviridovartem) [inline app/flask/models/tasks.py]: I think it's better rename to 'deleted', its more readable in future requests

File: app/flask/models/tasks.py [hunk 4]
```diff
@@ -27,26 +27,47 @@ def __init__(self, conn, cursor) -> None:
...
+
+    def get_deleted_tasks(self):
+        self.cursor.execute("SELECT * FROM Tasks WHERE is_deleted=true")
         return self.cursor.fetchall()
 
     def add_task(self, task):
         self.cursor.execute(
-            "INSERT INTO Tasks (NAME, COMMENT) VALUES (%s, %s);",
+            "INSERT INTO Tasks (NAME, COMMENT, IS_DELETED) VALUES (%s, %s, %s);",
+            (task["name"], task["comment"], False),
```
Comment (sviridovartem) [inline app/flask/models/tasks.py]: (task["name"], task["comment"], **False**),
by default deleted ==false, are you sure that we need to pass it here?

File: app/flask/models/tasks.py [hunk 5]
```diff
@@ -27,26 +27,47 @@ def __init__(self, conn, cursor) -> None:
...
+        )
+        self.conn.commit()
+
+    def delete_task(self, id):
+        self.cursor.execute(
+            "UPDATE Tasks SET is_deleted = (%s) WHERE id=%s",
```
Comment (sviridovartem) [inline app/flask/models/tasks.py]: you are using python 3.12.2, it means that f-strings are more readable

Comment (StanislavKharkiv) [reply app/flask/models/tasks.py]: done all

**Pull request merged**